classes/Http/Controller/TalkController.php

```diff
@@ -61,14 +61,18 @@ public function viewAction(Request $request)
         /* @var Speakers $speakers */
         $speakers = $this->service('application.speakers');
 
+        $talkId = (int) $request->get('id');
```

Comment:
While at it, I'm moving this out of the `try` block, as I don't see anything that could go wrong here.